### PR TITLE
Add GitHub page for docs

### PR DESCRIPTION
**Pull request by ciungulete**
- Add docs folder
- Add Github action to deploy pages

### CODE REVIEW

**Comment by sandervanhooft** [comment]
@ciungulete Something going wrong with the favicon:

image

**Comment by sandervanhooft** [comment]
(Latest Safari)

**Comment by sandervanhooft** [comment]
Can we add an "v2 alpha" label in the top nav, perhaps with an ℹ️ link?

"This is an alpha release, intended for development and testing purposes. This release is not yet ready for use in production."

**Comment by jelleroorda** [comment]
Hey @ciungulete, awesome work! This PR would close #262. Can you add one of the closing keywords in the PR so it would be closed automatically on merging? Thanks!

**Comment by sandervanhooft** [comment]
> Hey @ciungulete, awesome work! This PR would close #262. Can you add one of the closing keywords in the PR so it would be closed automatically on merging? Thanks!

We have our own process here, thanks :)

File: docs/01-instalation.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
+# Installation
```
Comment (sandervanhooft) [inline docs/01-instalation.md:1]: Can you rename this file to `01-installation.md`? There's an `l` missing

Comment (ciungulete) [reply docs/01-instalation.md:1]: Done

**Pull request closed without merging**